en/codegen/layout-codegen.md

```diff
@@ -12,56 +12,120 @@ For each selected Artboard or Group, a single Angular Component is created. Each
 
-The below two images are examples of what a developer would select for code generation.
+The below two images are examples of what a developer will select for code generation.
 
-In the below image the developer would select the Sketch Group `peopleEdit` for code generation. The other elements are chrome used by the designer to show the application and components in context.
+In the below image the developer selects the Sketch Group `peopleEdit` for code generation. The other elements are chrome used by the designer to show the application and components in context.
 
 <img class="responsive-img" src="../images/layout_codegen_people1.png" />
 
-In the below image the developer would select the Sketch Groups `peopleList` and `peopleDetails` for code generation, ignoring the navigation chrome elements.
+In the below image the developer will select the Sketch Groups `peopleList` and `peopleDetails` for code generation, ignoring the navigation chrome elements.
 
 <img class="responsive-img" src="../images/layout_codegen_people2.png" />
 
 ### Component Sizing
 
 > [!Note]
-> Artboards in Sketch do not have fixed height or fixed width settings so this section on Component Sizing only applies to a Component withing a Group that has been selected for code generation.
+> Artboards in Sketch do not have fixed height or fixed width settings so this section on Component Sizing only applies to a Component within a Group that the user selects for code generation.
 
 When a Group is selected for component code generation and the Components inside have a fixed height and or a fixed width, those fixed size values will be added to the generated component CSS.
-Normally Angular components are sized or resized by their parent at runtime. However, there are design scenarios where the designer wants the component to be a fixed size element when added to the Angular application. Code generation supports this design scenario.
-
-### After Component Code Generation – Before Rendering in The Browser
-
-Developers need to perform the following edits of generated components before rendering in the browser:
-
-- If the HTML has formControlName elements, the formGroup comment under the `<form>` tag must be uncommented: `<!-- [formGroup]="customerForm" -->`
-- In the TypeScript, check for `<!-- // TODO – uncomment-->` and uncomment the FormGroup comment if you have one.
-- In the TypeScript, check the `ngOnInit` method and uncomment the form code.
+Normally the parent of Angular components is responsible for their sizing or resizing at runtime. However, there are design scenarios where the designer wants the component to be a fixed size element when added to the Angular application. Code generation supports this design scenario.
 
 ## Sketch Element Names
 
 Code generation requires each element in the Sketch Layer List to have a name. Any element without a name will be skipped and not processed. This includes all of the skipped element's children as well.
```

Comment:
Not sure if this is still valid. We do not use the user defined names of the layers in the new codegen side anymore.

Reply:
Removing `## Sketch Element Names` section